changelog.json

```
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "1.6.0-beta1": [
+      "[New] Help users add their first repo during onboarding - #6474",
```

Sorry, missed this one.

```suggestion
      "[New] Help users add their first repository during onboarding - #6474",
```